gdnative/multi-lod/src/lod.h

```diff
@@ -42,8 +36,19 @@
 #include <MultiMeshInstance.hpp>
 #include <MultiMesh.hpp>
 
+// Doesn't currently support more than 4 LODs for meshes due to other hardcoded variables.
+#define LOD_COUNT 4
+
+#define PI 3.14159f
+
+#ifndef CLAMP
+#define CLAMP(m_a, m_min, m_max) (((m_a) < (m_min)) ? (m_min) : (((m_a) > (m_max)) ? m_max : m_a))
+#endif
+
 namespace godot {
 
+class LODBaseComponent;
```

Comment:
Can you add a short description/comment/title about what the purpose of the class is, like on the other ones?